themes/iceberg.color-theme.json

```diff
@@ -91,7 +90,7 @@
 		"list.hoverBackground": "#1e2132",
 		"list.warningForeground": "#e9b189",
 		"menu.background": "#1e2132",
-		"menu.foreground": "#6b7089",
+		"menu.foreground": "#c6c8d1A6",
```

Comment:
What about menu foreground colors? I'd make them opaque, too.

Reply:
Yes opaque color seems to be better. (also about `titleBar.activeForeground`)